### DISCUSSION TITLE
User dropdown in additional fields in workflow actional panel

### DISCUSSION THREAD

**Discussion by neha04** [pimcore/pimcore#8029] [Q&A / Support]
Hi
I am tring to add users select box in workflow action panel as given in documentaion .
additionalFields:
                                -
                                    name:                workflowUser
                                    title:               Select User
                                    setterFn:            setWorkflowUser
                                    fieldType:           user

The select box is added to the action panel but the select box does not have any options.

**Comment by dvesh3** [comment]
I'd classify this as a bug. please create an issue. thanks!

**Reply by neha04** [reply]
Sure.

**Reply by ghost** [reply]
@neha04 or @dvesh3 I am facing same issue, can u please guide, also I want to know like how to use optionProvider on workflow additionalFields.

Thanks.

**Reply by dvesh3** [reply]
@rushit-credencys this is not done yet. Issue is open with PR welcome status https://github.com/pimcore/pimcore/issues/8059 would you like to contribute?

**Reply by ghost** [reply]
@dvesh3  this is open with optionProvider also? can u pls confirm if u know?

It will help us to move on next step.

Thanks.

**Reply by dvesh3** [reply]
yes, I don't see anything in `getAdditionalFieldsForAction` which should set the options from optionProvider.

**Reply by ghost** [reply]
Thanks for your immediate replay @dvesh3.
